content/09.figures-tables.md

```diff
@@ -1,7 +1,11 @@
 ## Figures {.page_break_before}
 
-![Overview of the GitHub web interface...](images/figure-2.png){#fig:github}
+![An overview of the landing page for a GitHub repository. All files associated with the project are shown and accessible through the web interface (A).  ??? (B).  The repository has a toolbar with various ways to interact with the project including viewing code, posting issues, opening discussions, and projects (C) (see Box 1 for definitions). Contributors to the project are shown with links to their GitHub profiles (D).](images/github_diagram.png){#fig:github-diagram}
 
-<!--Here's a ref to that figure (Figure @fig:github).-->
+<!--Reference this figure in text like this (Figure @fig:github-diagram).-->
+
+![A summary of ways GitHub can be used showing the difficulty and degree of collaborativity for each. The vertical axis represents technical difficulty such that activities higher on the scale require knowledge and use of more GitHub features than activities lower on the scale.  Each activity spans a region of the horizontal axis representing who is involved with or benefits from each activity.  For example, storing data and code mainly benefits individuals or a lab group while making data and code citable and reproducible benefit other labs and the greater community as well.](images/scatterblob_1.png){#fig:scatterblob}
```

Comment:
@LunaSare, I'd love some input on this.  I just guessed!

Reply:
I think the caption looks great already, @Aariq!
Based on previous discussions with @robcrystalornelas and @emmajhudgins , I would add the following:

"A summary of ways GitHub can be used `in Ecology and Evolution,` showing ~~the~~ `technical` difficulty and degree of collaborativity for each. ~~The vertical axis represents technical difficulty such that~~ Activities higher on the vertical axis require `usage` knowledge ~~and use~~ of more GitHub features than activities lower on the scale.  `On the horizontal axis,` each activity spans a region ~~of the horizontal axis~~ representing who is `potentially` involved with or benefits from each activity.  For example, storing data and code mainly benefits individual `researchers` or `members of` a lab group`,` while making data and code citable and reproducible benefit other labs and the `larger` ~~greater~~ community as well.
`Independently of a users knowledge level of GitHub features, there are ways to use GitHub that allow tapping unto one of the most salient benefits of the platform: facilitating and enhancing collaboration.`